pkgs/applications/misc/pomodoro/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+let
+  pname = "pomodoro";
+  version = "0.1.0";
+in rustPlatform.buildRustPackage {
+  inherit pname;
+  inherit version;
+  src = fetchFromGitHub {
+    # Anna Aurora forked SanderJSA's repo to add git version tags. SanderJSA didn't respond when Anna Aurora requested dem to add git version tags.
+    owner = "auroraanna";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "ZA1q1YVJcdSUF9NTikyT3vrRnqbsu5plzRI2gMu+qnQ=";
```

```suggestion
    sha256 = "sha256-ZA1q1YVJcdSUF9NTikyT3vrRnqbsu5plzRI2gMu+qnQ=";
```
SRI hashes are only valid when they have their prefix.